ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/sdn2ovn-rollback/cucushift-installer-rehearse-aws-ipi-ovn-sdn2ovn-rollback-workflow.metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/ovn/sdn2ovn-rollback/cucushift-installer-rehearse-aws-ipi-ovn-sdn2ovn-rollback-workflow.yaml",
```

Comment:
The same comments as https://github.com/openshift/release/pull/32093#discussion_r965441288, for easy understanding of what is the target networktype, I suggest to move the workflow to `aws/ipi/sdn` folder.

Reply:
> The same comments as [#32093 (comment)](https://github.com/openshift/release/pull/32093#discussion_r965441288), for easy understanding of what is the target networktype, I suggest to move the workflow to `aws/ipi/sdn` folder.

@zhaozhanqi How about to keep sdn2ovn migration in aws/ipi/ovn and rollback in aws/ipi/sdn? WDYT?

Reply:
@jianlinliu Move the workflow to aws/ipi/sdn folder now.